doc/userguide/rules/transforms.rst

```diff
@@ -21,6 +21,36 @@ Example::
 
 .. note:: not all sticky buffers support transformations yet
 
+domain
+------
+
+Takes the buffer, and prepends a ``.`` character providing concise domain checks. For example,
+an input string of ``hello.google.com`` would be modified and become ``.hello.google.com``.
```

Comment:
Probably good to add that the dot is added to allow matching 'content:".google.com"; against just 'google.com' as well (which is transformed to '.google.com')

Reply:
Good suggestion ... will add.